app/views/tasks/collecting_events/parse/stepwise/lat_long/update.js.erb

```diff
@@ -0,0 +1,38 @@
+// j is an alias for escape_javascript
+var html;
+
+function startSpinner() {
+  timer.start();
+  setTimeout(stopSpinner, 5000);
+  $("#matching_span").mx_spinner('show');
+}
+
+function stopSpinner() {
+  timer.stop();
+  $("#matching_span").mx_spinner('hide');
+}
+
+html = $('<%= 'placekeeper' %>');
+startSpinner();
+
+var paging;
+//paging = $('<%#= j(render(partial: 'paging_data')) %>');
```

Comment:
Is this a feature waiting to happen, or abandoned?  Likely remove these comments.